### PR TITLE
Tabs permission as optional permission.

### LINKED ISSUE #64: Plugin requires too many permissions
With today’s upgrade, I’m seeing this box in the Chrome toolbar:

screen shot 2016-03-10 at 21 34 30

> **Notifier for GitHub requires new permissions**
> 
> The newest version of the extension "Notifier for
> GitHub" requires more permissions, so it has been
> disabled.
>
> It can now:
> - Read your browsing history

I don’t see any reason to allow this plugin to read my browsing history.

### PR DESCRIPTION
**Pull request by Narno**
Fixes #64.

### CODE REVIEW

**Comment by likerRr** [comment]
:+1: had to disable extension until this PR is accepted

File: extension/main.js [hunk 1]
```diff
@@ -90,16 +112,17 @@
...
-				chrome.tabs.update(tabs[0].id, ghTab);
-			} else if (tab.url === 'chrome://newtab/') {
-				chrome.tabs.update(null, ghTab);
-			} else {
-				chrome.tabs.create(ghTab);
-			}
-		});
+		// request optional permissions the 1rst time
+		if (window.GitHubNotify.settings.get('optional_permissions') === undefined) {
+			chrome.permissions.request({
```
Comment (sindresorhus) [inline extension/main.js]: No need to save the permission. Just `request` each time. Chrome will only ask once.

Comment (Narno) [inline extension/main.js]: Hum, are you sure about that? If I remember, the ```request``` method displays the permission pop-up each time.
I should test! :-)

Comment (Narno) [inline extension/main.js]: OK, after tests: Chrome will only ask once if user accepts the new permission.

Comment (Narno) [inline extension/main.js]: Ping @sindresorhus about my previous comment.

**Comment by sindresorhus** [comment]
Thanks @Narno :)

**Comment by Narno** [comment]
You're welcome! ;-)

**Comment by iamed2** [comment]
This is a really nice solution and I appreciate the effort!

**Pull request closed without merging**